.github/workflows/java-windows.yml

```
@@ -18,3 +18,5 @@ jobs:
                     java-version: ${{ matrix.java_version }}
             -   name: Build with Gradle
                 run: ./gr jvmBuild
+            -   name: build sample atrium+spek project
+                run: ./gr -p samples\jvm\spek build
```

Although this works as well we are not using the gradle version of the sample. Change to:
```suggestion
                run: samples\jvm\spek\gradlew -p samples\jvm\spek build
```

---

added